cross-region-replication-mm2-converter/src/main/java/com/amazonaws/services/crossregion/schemaregistry/kafkaconnect/CrossRegionReplicationMM2Converter.java

```diff
@@ -0,0 +1,170 @@
+package com.amazonaws.services.crossregion.schemaregistry.kafkaconnect;
+
+import com.amazonaws.services.schemaregistry.common.AWSSerializerInput;
+import com.amazonaws.services.schemaregistry.common.Schema;
+import com.amazonaws.services.schemaregistry.common.configs.GlueSchemaRegistryConfiguration;
+import com.amazonaws.services.schemaregistry.deserializers.GlueSchemaRegistryDeserializationFacade;
+import com.amazonaws.services.schemaregistry.deserializers.GlueSchemaRegistryDeserializerImpl;
+import com.amazonaws.services.schemaregistry.exception.AWSSchemaRegistryException;
+import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializationFacade;
+import com.amazonaws.services.schemaregistry.serializers.GlueSchemaRegistrySerializerImpl;
+import com.amazonaws.services.schemaregistry.utils.AWSSchemaRegistryConstants;
+
+import kotlinx.serialization.SerializationException;
+import lombok.Data;
+import lombok.Getter;
+
+import org.apache.kafka.connect.data.*;
+import org.apache.kafka.connect.errors.DataException;
+import org.apache.kafka.connect.storage.Converter;
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+@Data
+public class CrossRegionReplicationMM2Converter implements Converter {
+
+    private GlueSchemaRegistryDeserializationFacade deserializationFacade;
+    private GlueSchemaRegistrySerializationFacade serializationFacade;
+    private AwsCredentialsProvider credentialsProvider;
+    private GlueSchemaRegistryDeserializerImpl deserializer;
+    private GlueSchemaRegistrySerializerImpl serializer;
+    private boolean isKey;
+
+    /**
+     * Constructor used by Kafka Connect user.
+     */
+    CrossRegionReplicationMM2Converter(){};
+
+    /**
+     * Constructor accepting AWSCredentialsProvider.
+     *
+     * @param credentialsProvider AWSCredentialsProvider instance.
+     */
+    CrossRegionReplicationMM2Converter(
+            GlueSchemaRegistryDeserializationFacade deserializationFacade,
+            GlueSchemaRegistrySerializationFacade serializationFacade,
+            AwsCredentialsProvider credentialsProvider,
+            GlueSchemaRegistryDeserializerImpl deserializerImpl,
+            GlueSchemaRegistrySerializerImpl serializerImpl) {
+
+        this.deserializationFacade = deserializationFacade;
+        this.serializationFacade = serializationFacade;
+        this.credentialsProvider = credentialsProvider;
+        this.deserializer = deserializerImpl;
+        this.serializer = serializerImpl;
+    }
+
+    /**
+     * Configure the MM2 Schema Replication Converter.
```

Comment:
Not necessarily specific to MM2. We can remove the reference to MM2.

Reply:
Have renamed excluding the MM2 part